ci/requirements-dev-3.7.yml

```diff
@@ -0,0 +1,39 @@
+channels:
+  - conda-forge
+dependencies:
+  - click
+  - clickhouse-cityhash
+  - clickhouse-driver>=0.0.8
+  - clickhouse-sqlalchemy
+  - cmake
+  - flake8
+  - google-cloud-bigquery>=1.0.0
+  - graphviz
+  - impyla>=0.14.0
+  - jinja2
+  - lz4
+  - multipledispatch
+  - numpy
+  - pandas
+  - plumbum
+  - psycopg2
+  - pyarrow>=0.6.0
+  - pymapd>=0.3.2
+  - pymysql
+  - pytables
+  - pytest
+  - pytest-cov
+  - pytest-xdist
+  - python=3.7
+  - python-graphviz
+  - python-hdfs>=2.0.16
+  - pytz
+  - regex
+  - requests
+  - ruamel.yaml
+  - six
+  - sqlalchemy>=1.0.0,<1.1.15
```

Comment:
Packages that satisfy this constraint do not exist on `conda-forge`. After #1543 we'll be able to use `sqlalchemy` without constraints. There might still be additional work to do to get 3.7 working, we'll see.

Reply:
Guessed so :)